modules/AlphaWalletFoundation/AlphaWalletFoundation/Blockies/BlockieGenerator.swift

```diff
@@ -116,20 +104,20 @@ public class BlockiesGenerator {
     private func cacheBlockie(address: AlphaWallet.Address, blockie: BlockiesImage, size: BlockieSize) {
         switch size {
         case .sized(let size, let scale):
-            let key = BlockieKey(address: address, size: size, scale: scale)
+            let key = "\(address.eip55String)-\(size)-\(scale)"
```

Comment:
Why? Because it's simpler? It is :)

Reply:
yes, guess a bit easier, and less additional types